config/routes.rb

```diff
@@ -95,9 +95,9 @@
 
   resources :invitations, only: [:show, :create, :destroy], param: :invitation_slug do
     member do
-      post :accept, action: :update
-      post :decline, action: :update, decline: true
-      post :resend, action: :resend
+      get :accept
```

Comment:
It's standard to use non-get routes for any action that can change a resource.  Do we have a special use case for using get here?  I'd expect the accept and decline actions to use either patch or put, unless they aren't updating the invitations state.  If we have a good reason that's fine.  Otherwise, I'd follow the convention.